### ISSUE TITLE
Unable to rename from tmp to dst

### ISSUE THREAD

**Issue by mattfarina** [flavio/kuberlr#13]
Getting an error when attempting to rename the downloaded file. I think it has to do with the use of `os.Rename` which has limitations (e.g., it is unable to handle a source and destination on different drives).

**Comment by mattfarina** [comment]
The specific error I'm getting is on Windows (though it could happen elsewhere) and is...

```
Cross-device error trying to rename a file: rename C:\Users\Matt\AppData\Local\Temp\kuberlr-kubectl-116305899 C:\Users\Matt\.kuberlr\windows-amd64\kubectl1.17.17.exe: The process cannot access the file because it is being used by another process. -- will do a full copy
F0817 10:00:06.031214    8892 main.go:69] rename C:\Users\Matt\AppData\Local\Temp\kuberlr-kubectl-116305899 C:\Users\Matt\.kuberlr\windows-amd64\kubectl1.17.17.exe: The process cannot access the file because it is being used by another process.
```

**Comment by mattfarina** [comment]
Note, while I get the error it actually works. The kubectl version I'm looking for ends up in the cache.

**Comment by flavio** [comment]
The fix is part of the 0.4.1 release
